docker/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN \
       gnutls-devel \
       libgcrypt-devel \
       libcurl-devel \
+      openssl-devel \
```

Comment:
I guess that openssl-devel is installed as transitive depedency of any other, so probably this is not actually needed. However, better to be explicit.

(Added also in build_sources.md documentation)

NTC